lib/cli.js

```diff
@@ -53,7 +53,8 @@ module.exports = (process) => {
       npm.config.set('usage', false, 'cli')
     }
 
-    npm.updateNotification = await updateNotifier(npm)
+    npm.updateNotification = updateNotifier(npm).then(notification =>
+      npm.updateNotification = notification)
```

Comment:
```suggestion
    updateNotifier(npm).then(notification => {
      npm.updateNotification = notification
    })
```
why not wait to assign it at all until later, so you don't have a promise/non-promise property? that seems zalgoish

Reply:
Yeah, I guess I removed the bit where we await it in the reify-output.js (to ensure that we _always_ check for some actions that are known to be slow).  So that's fine.